include/TPP/Passes.td

```diff
@@ -43,6 +43,28 @@ def ConvertLinalgToFunc : Pass<"convert-linalg-to-func", "ModuleOp"> {
                            "linalg::LinalgDialect", "LLVM::LLVMDialect"];
 }
 
+def VectorizationPass : Pass<"vectorization-pass",
+                                     "func::FuncOp"> {
+  let summary = "convert linalg to memref/vector";
+  let description = [{
+    Vectorization pass to convert linalg to memref/vector instead of Xsmm.
+  }];
+
+  let dependentDialects = [ "memref::MemRefDialect", "linalg::LinalgDialect", "vector::VectorDialect" ];
+}
+
+
+def VectorContractPass : Pass<"vector-contract-pass",
```

Comment:
Do we ever want to not run these patterns?
I'd merge it with `VectorizationPass`.